### PR TITLE
Feature/save telemetry dashboard

### PR DESCRIPTION
**Pull request by cccs-jc**
script to generate dev superset config
    export/import cccs telemetry demo
    ipv4 regex accept ip that start with 0

### CODE REVIEW

File: superset-frontend/cccs-viz/cccs_create_dev_config.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+SQL_QUERY_MUTATOR = sql_query_mutator 
+SUPERSET_WEBSERVER_TIMEOUT = 100
+DEFAULT_EXPLORER_VIZ = 'hello_world'
+FEATURE_FLAGS = {
+    "ENABLE_TEMPLATE_PROCESSING": True,
+    "DASHBOARD_NATIVE_FILTERS": True,
+    "DASHBOARD_CROSS_FILTERS": True,
+    "DASHBOARD_NATIVE_FILTERS_SET": True,
+    "ENABLE_EXPLORE_DRAG_AND_DROP": True,
+    "DASHBOARD_RBAC": True,
```
Comment (cccs-joel) [inline superset-frontend/cccs-viz/cccs_create_dev_config.sh:25]: I know it does not hurt but the trailing comma should be removed.

File: superset-frontend/cccs-viz/cccs_export_dashboards.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+SOURCE_FILE=dashboards_export.json
+DEST_FILE=telemetry_dashboard.yaml
+DASHBOARDS="CCCS Telemery"
```
Comment (cccs-joel) [inline superset-frontend/cccs-viz/cccs_export_dashboards.sh:5]: Typo: Teleme>t<ry
```suggestion
DASHBOARDS="CCCS Telemetry"
```

File: superset-frontend/cccs-viz/cccs_export_dashboards.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+echo "Exporting dashboards"
+docker exec -it superset_app superset export-dashboards --dashboard-file /app/$SOURCE_FILE
```
Comment (cccs-joel) [inline superset-frontend/cccs-viz/cccs_export_dashboards.sh:8]: How/where do you assign the running Superset pod name to superset_app?

**Review by cccs-joel — CHANGES_REQUESTED** [review]

File: superset-frontend/cccs-viz/cccs_create_dev_config.sh (hunk 1 above)
Comment (cccs-jc) [reply superset-frontend/cccs-viz/cccs_create_dev_config.sh:25]: this is how its done all over in the config.py

TALISMAN_CONFIG = {
    "content_security_policy": None,
    "force_https": True,
    "force_https_permanent": False,
}

File: superset-frontend/cccs-viz/cccs_export_dashboards.sh (hunk 2 above)
Comment (cccs-jc) [reply superset-frontend/cccs-viz/cccs_export_dashboards.sh:5]: I'll fix that in the next pull request, I have a bunch of other changes on the go.

File: superset-frontend/cccs-viz/cccs_create_dev_config.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```
Comment (cccs-rc) [inline superset-frontend/cccs-viz/cccs_create_dev_config.sh:2]: Would love some doc explaining overall what this script's purpose is!

File: superset-frontend/cccs-viz/cccs_create_dev_config.sh [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+read -p 'Enter path hogwarts-superset-deployment/config/sql_query_mutator.py file: ' SQL_QUERY_MUTATOR_FILE
+
+echo "Adding $SQL_QUERY_MUTATOR_FILE content"
+echo "to $CONFIG_FILE"
+cat $SQL_QUERY_MUTATOR_FILE > $CONFIG_FILE
+
+echo "Adding dev configs"
+cat >> $CONFIG_FILE <<'_EOF'
+SQL_QUERY_MUTATOR = sql_query_mutator 
+SUPERSET_WEBSERVER_TIMEOUT = 100
```
Comment (cccs-rc) [inline superset-frontend/cccs-viz/cccs_create_dev_config.sh:17]: Is this a deliberate increase from the default? Were you running into timeouts? Just curious.

File: superset-frontend/cccs-viz/cccs_export_dashboards.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```
Comment (cccs-rc) [inline superset-frontend/cccs-viz/cccs_export_dashboards.sh:2]: A little bit of doc would be much appreciated!

File: superset-frontend/cccs-viz/cccs_import_dashboards.sh [hunk 7]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DASHBOARD_FILE=telemetry_dashboard.yaml
+
+echo "Copying dashboard file into docker container"
+docker cp $DASHBOARD_FILE superset_app:/app/$DASHBOARD_FILE
+
+echo "Imporing cccs dashboards"
```
Comment (cccs-rc) [inline superset-frontend/cccs-viz/cccs_import_dashboards.sh:8]:
```suggestion
echo "Importing cccs dashboards"
```

File: superset-frontend/cccs-viz/telemetry_dashboard.yaml [hunk 8]
```diff
@@ -0,0 +1,1000 @@
+{"dashboards":[
+{
+  "__Dashboard__": {
+    "css": "",
+    "dashboard_title": "CCCS Telemery",
```
Comment (cccs-rc) [inline superset-frontend/cccs-viz/telemetry_dashboard.yaml:5]:
```suggestion
    "dashboard_title": "CCCS Telemetry",
```

**Review by cccs-rc — CHANGES_REQUESTED** [review]

File: superset-frontend/cccs-viz/cccs_create_dev_config.sh (hunk 5 above)
Comment (cccs-jc) [reply superset-frontend/cccs-viz/cccs_create_dev_config.sh:17]: I was not running into timeouts, the default is 60?

Comment (cccs-rc) [reply superset-frontend/cccs-viz/cccs_create_dev_config.sh:26]: Yeah I believe so. Could just leave this line out if the default's good, but not a big deal.

**Review by cccs-rc — APPROVED** [review]

**Review by cccs-joel — APPROVED** [review]

**Pull request merged**